### PR TITLE
Shopify updates

### PR DESCRIPTION
**Pull request by bre-fitzgerald**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [x] Yes (**5/30**)
- [ ] No

### CODE REVIEW

**Comment by bre-fitzgerald** [comment]
@lydia-xie this is ready to be merged on **Friday, 5/27**. After merging, please create an ad hoc deploy and tag me for review, and I'll get it merged on Monday. Thanks!!

File: _docs/_partners/message_orchestration/channel_extensions/ecommerce/shopify/shopify.md [hunk 1]
```diff
@@ -275,6 +535,22 @@ Braze will map the supported Shopify data to user profiles using the customer's
 ## Using Shopify data in Braze
 Once you've completed your integration, take a look at our next Shopify [article]({{site.baseurl}}/partners/message_orchestration/channel_extensions/ecommerce/shopify/use_cases/) to learn how to use Shopify data in Braze for personalization and segmentation in your campaigns and Canvases.
 
+## Shopify advanced settings
+
+### Update abandoned checkout delay
+
+By default, Braze will automatically set the delay to trigger the `shopify_abandoned_checkout` event to 1 hour of inactivity. You can set the **Abandoned Checkout Delay** from 5 minutes up to 24 hours by selecting the dropdown and then selecting **Set Delay** on the Shopify partner page.
```
Comment (lydia-xie) [inline _docs/_partners/message_orchestration/channel_extensions/ecommerce/shopify/shopify.md:542]:
```suggestion
By default, Braze will automatically set the delay to trigger the `shopify_abandoned_checkout` event to one hour of inactivity. You can set the **Abandoned Checkout Delay** from 5 minutes up to 24 hours by selecting the dropdown and then selecting **Set Delay** on the Shopify partner page.
```
Minor suggestion for number formatting if we're referring to the abandonment default value but not the actual option like this sentence from earlier in the doc:

> The abandonment default is set to **1 hour** but is configurable within the **Advanced Settings** section on the Shopify partner page.

File: _docs/_partners/message_orchestration/channel_extensions/ecommerce/shopify/shopify.md [hunk 2]
```diff
@@ -74,9 +70,14 @@ Once the app installation is complete, Braze automatically creates your webhook
...
 | --- | --- | --- |
-| `shopify_abandoned_checkout` | [Custom Event]({{site.baseurl}}/user_guide/data_and_analytics/custom_data/custom_events/) | Shopify checkout updates webhook's trigger when a customer adds or removes items from their cart AND proceeds further into the checkout process including adding their personal information.<br><br>Braze will listen to the inbound Shopify checkout update webhooks and trigger the `shopify_abandoned_checkout` custom event when that checkout is considered abandoned after **1 hour** of checkout or cart activity. |
+| `shopify_abandoned_checkout` | [Custom Event]({{site.baseurl}}/user_guide/data_and_analytics/custom_data/custom_events/) | Shopify checkout updates webhook's trigger when a customer adds or removes items from their cart AND proceeds further into the checkout process including adding their personal information.<br><br>Braze will listen to the inbound Shopify checkout update webhooks and trigger the `shopify_abandoned_checkout` custom event when that checkout is considered abandoned. The abandonment default is set to **1 hour** but is configurable within the **Advanced Settings** section on the Shopify partner page. |
```
Comment (lydia-xie) [inline _docs/_partners/message_orchestration/channel_extensions/ecommerce/shopify/shopify.md:73]:
```suggestion
| `shopify_abandoned_checkout` | [Custom Event]({{site.baseurl}}/user_guide/data_and_analytics/custom_data/custom_events/) | Shopify checkout updates a webhook's trigger when a customer adds or removes items from their cart AND proceeds further into the checkout process, including adding their personal information.<br><br>Braze will listen to the inbound Shopify checkout update webhooks and trigger the `shopify_abandoned_checkout` custom event when that checkout is considered abandoned. The abandonment default is set to **1 hour** but is configurable within the **Advanced Settings** section on the Shopify partner page. |
```
Missing an article here, I think?

**Review by lydia-xie — CHANGES_REQUESTED** [review]
looks great! left two small comments :)

**Review by lydia-xie — APPROVED** [review]
lgtm!

**Comment by PAMILER** [comment]
Hi there did you want to add new section and images to your store I'll design a brand image's for store and section
  I can build a theme from scratch

**Pull request merged**